packages/lesswrong/components/sunshineDashboard/SunshineGoogleServiceAccount.tsx

```diff
@@ -38,7 +38,7 @@ const SunshineGoogleServiceAccount = ({ classes }: {
 
   const shouldWarn = !estimatedExpiry || (new Date(estimatedExpiry).getTime() - Date.now()) < WARN_THRESHOLD
 
-  if (loading || !userIsAdminOrMod(currentUser) || !hasGoogleDocImportSetting.get() || !shouldWarn) {
+  if (loading || !userIsAdmin(currentUser) || !hasGoogleDocImportSetting.get() || !shouldWarn) {
```

Comment:
Because mods don't have read access on `estimatedExpiry`, it appears to be null for them which causes the error message to appear even when the account is set up